### PR TITLE
Add calendar widget component

### PR DESCRIPTION
**Pull request by mikeFam**
**Issue:**
#16

**Solution:**
- Implemented the calendar widget 
- Uses the `react-calendar` library 

**Test Method:** 
- Will create a follow-up PR

**Risk:**
- Untested

**Reviewed By:**

### CODE REVIEW

File: client/src/components/CalendarWidget/EventView/index.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles } from "@material-ui/core/styles";
+// import Button from "@material-ui/core/Button";
```
Comment (Denijar) [inline client/src/components/CalendarWidget/EventView/index.js:4]: Delete this commented code

File: client/src/components/CalendarWidget/EventView/index.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        background: "#30A0F5",
+        border: 0,
+        borderRadius: 32,
+        color: "white",
+        height: 44,
+        padding: "5px 10px",
+    },
+}));
+
+export default function EventView({ selected, events, onRemoveEvent }) {
```
Comment (Denijar) [inline client/src/components/CalendarWidget/EventView/index.js:73]: nit: in the rest of the project we have the `export default EventView;` line just after the function. Might be worth checking this for all these files so we are staying consistent with the rest of the project.

File: client/src/components/CalendarWidget/index.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import Calendar from "react-calendar";
```
Comment (Denijar) [inline client/src/components/CalendarWidget/index.js:2]: I am unable to resolve this module when I run `npm start`. Was `react-calendar` supposed to be included in the `package.json`?

File: client/src/components/CalendarWidget/initial-events.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+/**
```
Comment (Denijar) [inline client/src/components/CalendarWidget/initial-events.js:1]: Are these dummy events meant to stay here? Perhaps they could be changed to something more meaningful for the demonstration such as some public holidays for example

**Review by Denijar — CHANGES_REQUESTED** [review]

File: client/src/components/CalendarWidget/initial-events.js (hunk 4 above)
Comment (danielyeom) [reply client/src/components/CalendarWidget/initial-events.js:1]: In the process of removing these

**Review by Denijar — APPROVED** [review]
Good effort to get this done in time for the demo. Thanks for not giving up on it :)))

**Review by dikunong — APPROVED** [review]
LGTM, your work to get it done through all the git troubles is appreciated :)

**Pull request merged**